templates/components/news-banner/news-banner.ui_patterns.yml

```diff
@@ -23,7 +19,7 @@ news-banner:
       preview: "This is the headline"
   use: "@news/news-banner/news-banner.twig"
   libraries:
-    - card:
+    - banner:
         css:
           component:
             ../../../dist/css/news-banner.css: { weight: 1000 }
```

Comment:
Remove the weight, please.

Reply:
@jenbreese Can you remove the weights in the ui_pattern.yml files and confirm that the css is still being applied correctly. Thank you.